app/models/piece.rb

```diff
@@ -18,6 +18,27 @@ def move!(x,y)
     end
   end
 
+  #Inital Capture Logic:
+  #Need game model to test id of game
+
+  #Set initial captured status to false
+  def default_captured_status
+    self.captured = false if self.captured = nil
+  end
+
+  #Find the target piece in the database base on the x,y coordinates
+  def capturable?(x,y)
+    Piece.where(:x_position => x, :y_position => y, :captured => false).where.not(:color => color).present?
+  end
+
+  #When the target is captured, change it's attributes and remove it from the board
+  def capture!(x,y)
+    opponent_piece = Piece.where(:x_position => x, :y_position => y)
+    if capturable?(x,y)
+      opponent_piece[0].update_attributes(:x_position => nil, :y_position => nil, :captured => true)
+    end
+  end
+
```

Comment:
I would expect capturable? to be called in the move method, not here.  Captures are going to initiate from a player's desire to move a piece to an x,y coordinate, thus triggering the move method.